EligibilityCheck/handler.ts

```diff
@@ -24,26 +23,25 @@ type IEligibilityCheckHandler = (
   context: Context,
   fiscalCode: FiscalCode
 ) => Promise<
-  // tslint:disable-next-line: max-union-size
   | IResponseSuccessRedirectToResource<InstanceId, InstanceId>
   | IResponseSuccessAccepted
   | IResponseErrorInternal
-  | IResponseErrorConflict
 >;
 
 initTelemetryClient();
 
 export function EligibilityCheckHandler(): IEligibilityCheckHandler {
   return async (context, fiscalCode) => {
     const client = df.getClient(context);
-    const status = await client.getStatus(fiscalCode);
-    if (status.runtimeStatus === df.OrchestrationRuntimeStatus.Running) {
-      return ResponseErrorConflict("Orchestrator already running");
+    const status = await client.getStatus(`${fiscalCode}-BV01DSU`);
```

Comment:
`${fiscalCode}-BV01DSU`)` should be a method

`export const getEligibilityCheckInstanceId (fiscalCode: FiscalCode) = `${fiscalCode}-BV01DSU``

since it's used here and there or at least define `BV01DSU` as a shared constant and 

(avoid to repeat it to avoid typos)